cli/main.go

```diff
@@ -68,8 +68,11 @@ func newLogger() logr.Logger {
 
 func main() {
 	var globalImpl bool
+	var logLevel string
 
 	flag.BoolVar(&globalImpl, "global-impl", false, "Record telemetry from the OpenTelemetry default global implementation")
+	flag.StringVar(&logLevel, "log-level", "info", "Define log visibility level")
```

Comment:
```suggestion
	flag.StringVar(&logLevel, "log-level", "", "Define log visibility level")
```

I think we should not set the default value here as info. Doing so will cause `OTEL_LOG_LEVEL=DEBUG` to be ignored since we add `WithLogLevel` after `WhitEnv` this is expected, but from a user who is running the agent image point of view this is not expected.
In addition we set the default level to info in `newInstConfig` if it is not define but env or by option.

Reply:
Yeah, makes sense what you say, changed to use like you recommended.

I have already set info as default on `newInstConfig`.